src/index.tsx

```diff
@@ -57,25 +58,27 @@ const Router = isBrowserRouterEnabled() ? BrowserRouter : HashRouter
 createRoot(container).render(
   <StrictMode>
     <Provider store={store}>
-      <FeatureFlagsProvider>
-        <QueryClientProvider client={queryClient}>
-          <Router>
-            <LanguageProvider>
-              <Web3Provider>
-                <ApolloProvider client={apolloClient}>
-                  <BlockNumberProvider>
-                    <Updaters />
-                    <ThemeProvider>
-                      <ThemedGlobalStyle />
-                      <App />
-                    </ThemeProvider>
-                  </BlockNumberProvider>
-                </ApolloProvider>
-              </Web3Provider>
-            </LanguageProvider>
-          </Router>
-        </QueryClientProvider>
-      </FeatureFlagsProvider>
+      <PersistGate loading={null} persistor={persistor}>
```

Comment:
This delays the first render until the state has been read, which is not what we want, because then the wallet connection attempt occurs, and overall it just adds jank to the initial render.

Would redux-persist still work without PersistGate? It's ok to render initially with empty state, so long as it re-renders after first load.

(Sorry - I know I initially suggested this, but it adds a lot of time to the initial render so I don't think it can go in.)